tests/data/output/enums/constant/output-json-schema.json

```diff
@@ -58,6 +58,8 @@
         },
         "port": {
             "type": "integer",
+            "minimum": 0,
```

Comment:
does it make sense to emit this in the default case ?

Reply:
IMHO `type: port` should always emit `min` / `max` even if there's `enum` with fixed values. This does contradict with my as small as possible JSON Schema, but in this case, I'd rather emit it.

Reply:
Alrighty, let's have it and if people have objections while using it later we can address them one by one and if not then brilliant